utils/utils.go

```diff
@@ -138,6 +138,58 @@ func HandlePanic() {
 	}
 }
 
+// GenerateGithubActions generates a GitHub Actions workflow file for Keploy
+func GenerateGithubActions(logger *zap.Logger, path string, appCmd string) {
+	// Determine the path based on the alias "keploy"
+	var keployPath string
+	isDockerCmd := len(os.Getenv("IS_DOCKER_CMD")) > 0
+	if isDockerCmd {
+		logger.Info("Running in docker environment, skipping the path determination")
+		keployPath = "./"
+	} else {
+		logger.Info("Determining the path of the keploy binary based on the alias \"keploy\"")
+		keployPath = "/usr/local/bin/keploybin" // Default path
+		aliasCmd := exec.Command("which", "keploy")
+		aliasOutput, err := aliasCmd.Output()
+		if err == nil && len(aliasOutput) > 0 {
+			keployPath = strings.TrimSpace(string(aliasOutput))
+		}
+		logger.Info("Path of the keploy binary determined successfully", zap.String("path", keployPath))
+	}
+	// Define the content of the GitHub Actions workflow file
+	actionsFileContent := `name: Keploy
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Test-Report
+        uses: keploy/testgpt@main
+        with:
+          working-directory: ./
+          keploy-path: ` + keployPath + `
```

Comment:
This is a github actions workflow, here keploy will not be automatically installed in the required destination, you will need to install everything again.